docs/data-sources/zookeeper_cluster_v2_instance.md

```diff
@@ -0,0 +1,169 @@
+*___two_factor_delete___*<br>
+<ins>Type</ins>: nested block, read-only, see [two_factor_delete](#nested--two_factor_delete) for nested schema<br>
+<br>
+*___current_cluster_operation_status___*<br>
+<ins>Type</ins>: string, read-only<br>
+<ins>Constraints</ins>: allowed values: [ `NO_OPERATION`, `OPERATION_IN_PROGRESS`, `OPERATION_FAILED` ]<br><br>Indicates if the cluster is currently performing any restructuring operation such as being created or resized<br><br>
+<a id="nested--data_centre"></a>
+## Nested schema for `data_centre`
+List of data centre settings.<br>
+### Read-only attributes
+*___cloud_provider___*<br>
+<ins>Type</ins>: string, read-only<br>
+<ins>Constraints</ins>: allowed values: [ `AWS_VPC`, `GCP`, `AZURE`, `AZURE_AZ` ]<br><br>Name of the cloud provider service in which the Data Centre will be provisioned.<br><br>
+*___number_of_nodes___*<br>
+<ins>Type</ins>: integer, read-only<br>
+<br>Total number of Zookeeper nodes in the Data Centre. Must be a multiple of `replicationFactor`.<br><br>
+*___replication_factor___*<br>
+<ins>Type</ins>: integer, read-only<br>
+<br>Number of racks to use when allocating nodes.<br><br>
+*___region___*<br>
+<ins>Type</ins>: string, read-only<br>
+<br>Region of the Data Centre. See the description for node size for a compatible Data Centre for a given node size.<br><br>
+*___client_to_cluster_encryption___*<br>
+<ins>Type</ins>: boolean, read-only<br>
+<br>Enables Client ⇄ Node Encryption.<br><br>
+*___status___*<br>
+<ins>Type</ins>: string, read-only<br>
+<br>Status of the Data Centre.<br><br>
+*___azure_settings___*<br>
+<ins>Type</ins>: nested block, read-only, see [azure_settings](#nested--azure_settings) for nested schema<br>
+<br>Azure specific settings for the Data Centre. Cannot be provided with AWS or GCP settings.<br><br>
+*___gcp_settings___*<br>
+<ins>Type</ins>: nested block, read-only, see [gcp_settings](#nested--gcp_settings) for nested schema<br>
+<br>GCP specific settings for the Data Centre. Cannot be provided with AWS or Azure settings.<br><br>
+*___id___*<br>
+<ins>Type</ins>: string, read-only<br>
+<br>ID of the Cluster Data Centre.<br><br>
+*___tag___*<br>
+<ins>Type</ins>: repeatable nested block, read-only, see [tag](#nested--tag) for nested schema<br>
+<br>List of tags to apply to the Data Centre. Tags are metadata labels which  allow you to identify, categorize and filter clusters. This can be useful for grouping together clusters into applications, environments, or any category that you require.<br><br>
+*___name___*<br>
+<ins>Type</ins>: string, read-only<br>
+<br>A logical name for the data centre within a cluster. These names must be unique in the cluster.<br><br>
+*___nodes___*<br>
+<ins>Type</ins>: repeatable nested block, read-only, see [nodes](#nested--nodes) for nested schema<br>
+<br>
+*___node_size___*<br>
+<ins>Type</ins>: string, read-only<br>
+<br>Size of the nodes provisioned in the Data Centre. --AVAILABLE_NODE_SIZES_MARKER_V2_ZOOKEEPER--<br><br>
```

Comment:
These one should be `AVAILABLE_NODE_SIZES_MARKER_V2_APACHE_ZOOKEEPER`, the `BundleType` for zookeeper. Then it will be replaced by the available node sizes for that bundle while the docs are being generated.

Reply:
Seems this is generated from the source yaml file and since i didn't see this comment and that file has been merged I couldn't change it now.